Login.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { login } from './api';
+
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const token = await login(email, password);
+    if (token) {
+      localStorage.setItem('token', token);
+      history.push('/dashboard');
```

Comment:
Storing tokens in local storage can be a security risk. Consider using httpOnly cookies or other secure ways to store tokens.